packages/rum/src/rum.entry.ts

```diff
@@ -57,7 +57,7 @@ const STUBBED_RUM = {
 
 export type RumGlobal = typeof STUBBED_RUM
 
-export const datadogRum = makeGlobal(STUBBED_RUM)
+export let datadogRum = makeGlobal(STUBBED_RUM)
```

Comment:
```suggestion
export const datadogRum = makeGlobal(STUBBED_RUM)
```
it does not seem necessary

Reply:
Indeed, I changed the assign implementation and forgot to revert this change !